src/Microsoft.VisualBasic.Core/tests/FinancialTests.cs

```diff
@@ -3,13 +3,14 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
-using System.Collections.Generic;
 using Xunit;
 
 namespace Microsoft.VisualBasic.Tests
 {
     public class FinancialTests
     {
+        private static bool IsNotArmOrAlpine() => !PlatformDetection.IsArmProcess && !PlatformDetection.IsAlpine;
```

Comment:
This won't work for ARM64. You would need `IsArmOrArm64Process`. 

As much as we call 32 bit ARM "ARM32" I think it is not officially called that so it gets confusing.